menu-bottom.js

```diff
@@ -2,15 +2,34 @@ const mainMenu = document.querySelector('.mainMenu');
 const closeMenu = document.querySelector('.closeMenu');
 const openMenu = document.querySelector('.openMenu');
 
-openMenu.addEventListener('click',show);
-closeMenu.addEventListener('click',close);
-mainMenu.addEventListener('click', close);
-
 function show() {
-    mainMenu.style.display = 'flex';
-    mainMenu.style.top = '0';
+  mainMenu.style.display = 'flex';
+  mainMenu.style.top = '0';
 }
 
 function close() {
-    mainMenu.style.top = '-100%'
+  mainMenu.style.top = '-100%';
+}
+
+openMenu.addEventListener('click', show);
+closeMenu.addEventListener('click', close);
+mainMenu.addEventListener('click', close);
+
+const popUp = document.querySelector('.popup');
+const openModal = document.querySelector('.button-project');
+const closeModal = document.querySelector('.closePop');
+const overlay = document.querySelector('#overlay');
+
+function showPop() {
+  popUp.style.display = 'flex';
+  popUp.style.top = '0';
+  overlay.classList.add('active');
 }
+
+function closePop() {
+  popUp.style.top = '-150%';
+  overlay.classList.remove('active');
+}
+
+openModal.addEventListener('click', showPop);
+closeModal.addEventListener('click', closePop);
```

Comment:
- [ ]  It may be a good idea to fix your JavaScript file. Following the requirements, you should implement an array that stores objects which store the information of your projects. Example:
```javascript
const projects = [
{
    name: "Project 1",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image1.jpg",
    technologies: ["Phaser3", "HTML", "Jest"],
    link: "link_to_live_version1.com",
},
{
    name: "Project 2",
    description: "A Phaser2 plataformer game built as part of the Microverse.",
    image: "link_to_image2.jpg",
    technologies: ["Phaser3", "JavaScript", "CSS"],
    link: "link_to_live_version2.com",
}
]
```
And use them to create your project cards and send those to your HTML(You could do it with [Resource 1](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML), [Resource 2](https://www.w3schools.com/jsref/met_node_appendchild.asp)).

![Project cards](https://user-images.githubusercontent.com/79876959/141345965-0a78ca92-5b1f-4d3f-b9d2-88f15491e02b.PNG)

It means that already created HTML code for the projects list should be removed and instead it should be added via JavaScript. It will allow you to create projects cards dynamically. Don't forget that your function for creating those cards in JavaScript needs to run when the page loads. 

After that, you can reuse this info to populate your popup window.  

Some resources to do it:
- [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event)
- [onload](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload)